update_interfaces.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# 1. ensure api-interfaces is populated w latest artifacts
+#git pull --recurse-submodules
+git submodule update --remote --init --recursive
+
+# 2. overwrite local artifacts for this language
```

Comment:
This script largely overlaps with the CI, if we want submodule updates to be manual (which is probably likely), then that is really the only step needed here.  The other steps should be handled by the CI.

Reply:
The plan might be that a push to api-interfaces triggers a build of the sdk, then it makes sense.

Reply:
this script actually doesn't run at all, i should just remove it. I wrote the script out and then as I started building the action, I decided to just move the shell commands to CI steps.